ci/eval/diff.nix

```
@@ -18,13 +18,22 @@ let
       added: [ <keys only in the second object> ],
       removed: [ <keys only in the first object> ],
       changed: [ <keys with different values between the two objects> ],
+      rebuilds: [ <keys in the second object with values not present at all in first object> ],
     }
   */
   diff =
+    old: new:
     let
       filterKeys = cond: attrs: lib.attrNames (lib.filterAttrs cond attrs);
+      oldOutputs = lib.pipe old [
+        (lib.mapAttrsToList (_: lib.attrValues))
+        lib.flatten
+        lib.unique
+        (map (lib.flip lib.nameValuePair true))
+        lib.listToAttrs
```

Would this be simpler with `genAttrs`?

```suggestion
        (lib.flip lib.genAttrs (_: true))
```